Networking/Networking/Requests/JetpackRequest.swift

```diff
@@ -83,7 +83,15 @@ private extension JetpackRequest {
     ///
     var dotcomMethod: HTTPMethod {
         // If we are calling DELETE via a tunneled connection, use GET instead (DELETE will be added to the `_method` query string param)
-        return method == .delete ? .get : method
+        // Likewise, PUT requests should be sent as POST for the tunneled request
```

Comment:
just wondering, is there documentation on this behavior? if so, it might be helpful adding a comment here 🙂

Reply:
I think I read that somewhere but cannot find it now 😓 I can only find it implicitly indicated in the Postman file referenced in P91TBi-3Qb-p2. My searching skill really needs to be improved!